apps/frontend/src/pages/access-management/libs/components/group-users-table/group-users-table.tsx

```diff
@@ -43,7 +43,7 @@ const GroupUsersTable = ({
 	setValue,
 }: Properties): JSX.Element => {
 	const dispatch = useAppDispatch();
-	const { onSearch, search } = useSearch();
+	const { onSearch, search } = useSearch({ shouldUseQuery: false });
```

Comment:
let's rename it to isSavedToUrl
```suggestion
	const { onSearch, search } = useSearch({ isSavedToUrl: false });
```